packages/blockchain-extension/test/commands/UserInputUtil.test.ts

```diff
@@ -62,6 +62,8 @@ describe('UserInputUtil', () => {
     let walletEntryTwo: FabricWalletRegistryEntry;
     let environmentEntryOne: FabricEnvironmentRegistryEntry;
     let environmentEntryTwo: FabricEnvironmentRegistryEntry;
+    let opsEnvironment1: FabricEnvironmentRegistryEntry;
+    let opsEnvironment2: FabricEnvironmentRegistryEntry;
```

Comment:
These are used only in one test and not set at before each, so they should be created as part of the test itself.